src/Symfony/Component/Runtime/README.md

```diff
@@ -0,0 +1,13 @@
+Runtime Component
+=================
+
+Symfony Runtime enables decoupling your apps from global state.
+
+Resources
+---------
+
+  * [Documentation](https://symfony.com/doc/current/components/runtime.html)
```

Comment:
should we include a "Getting Started" section instead? i believe component docs are on a diet :)